### PR TITLE
OKD-103: Build the machine-os-images image for OKD/SCOS

### PR DESCRIPTION
**Pull request by LorbusChris**
Also fixes: https://github.com/okd-project/okd/issues/1506
/cc @vrutkovs @derekhiggins

### CODE REVIEW

**Review by vrutkovs — APPROVED** [review]

**Comment by vrutkovs** [comment]
You need to run `make jobs` to complete metadata

**Comment by vrutkovs** [comment]
`make ci-operator-config` required

**Review by vrutkovs — APPROVED** [review]

File: ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.12__okd-scos.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+images:
+- build_args:
+  - name: DIRECT_DOWNLOAD
+    value: "true"
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_4.12_installer:
+      as:
+      - registry.ci.openshift.org/ocp/4.12:installer
```
Comment (vrutkovs) [inline ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.12__okd-scos.yaml:21]:
```suggestion
      - registry.ci.openshift.org/origin/scos-4.12:installer
```

File: ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.13__okd-scos.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+images:
+- build_args:
+  - name: DIRECT_DOWNLOAD
+    value: "true"
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_4.13_installer:
+      as:
+      - registry.ci.openshift.org/ocp/4.13:installer
```
Comment (vrutkovs) [inline ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.13__okd-scos.yaml:21]:
```suggestion
      - registry.ci.openshift.org/origin/scos-4.13:installer
```

File: ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.14__okd-scos.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+images:
+- build_args:
+  - name: DIRECT_DOWNLOAD
+    value: "true"
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_4.13_installer:
+      as:
+      - registry.ci.openshift.org/ocp/4.13:installer
```
Comment (vrutkovs) [inline ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.14__okd-scos.yaml:21]:
```suggestion
      - registry.ci.openshift.org/origin/scos-4.13:installer
```

File: ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.13__okd-scos.yaml (hunk 2 above)
Comment (LorbusChris) [reply ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.13__okd-scos.yaml:21]: Doesn't this replace https://github.com/openshift/machine-os-images/blob/main/Dockerfile#L1 and is correct here?

**Pull request merged**